### PR TITLE
Update windows-cheatsheet.md

### PR DESCRIPTION
**Pull request by aniketsinha2002**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/windows-cheatsheet.md [hunk 1]
```diff
@@ -68,6 +69,14 @@ created: 2022-10-20
 |` Win + Tab`| Open task view|
 |` Win + M`| Minimize all windows |
 |` Win + R`| Open the Run dialog box |
+|` Win + A`| Open the Action Center |
+|` Win + C`| Launch Cortona in listening mode (used for voice commands) |
+|` Win + S`| Search/Launch Cortona (listening mode) |
+|` Win + I`| Open the settings app |
+|` Win + Ctrl + D`| Create New Virtual Desktop |
+|` Win + Ctrl + F4`| Close the Current Desktop |
+|` Win + Ctrl + Left Arrow Key / Right Arrow Key`| Switch between Virtual Desktops |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:78]:
```suggestion
| `Win + Ctrl + Left Arrow Key` <br /> `Win + Ctrl + Right Arrow Key` | Switch between Virtual Desktops |
```
| Command | Description |
| ----------- | --------------|
| `Win + Ctrl + Left Arrow Key` <br /> `Win + Ctrl + Right Arrow Key` | Switch between Virtual Desktops |

Please, modify it like this.

File: CheatSheets/windows-cheatsheet.md [hunk 2]
```diff
@@ -68,6 +69,14 @@ created: 2022-10-20
...
+|` Win + Left Arrow Key / Right Arrow Key`| Snap Current Windows to left or right |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:79]:
```suggestion
| `Win + Left Arrow Key` <br /> `Win + Right Arrow Key` | Snap Current Windows to left or right |
```
| Command | Description |
| ------------ | ------------ |
| `Win + Left Arrow Key` <br /> `Win + Right Arrow Key` | Snap Current Windows to left or right |

Please, modify it like this.

File: CheatSheets/windows-cheatsheet.md [hunk 3]
```diff
@@ -68,6 +69,14 @@ created: 2022-10-20
 |` Win + Tab`| Open task view|
 |` Win + M`| Minimize all windows |
 |` Win + R`| Open the Run dialog box |
+|` Win + A`| Open the Action Center |
+|` Win + C`| Launch Cortona in listening mode (used for voice commands) |
+|` Win + S`| Search/Launch Cortona (listening mode) |
+|` Win + I`| Open the settings app |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:75]:
```suggestion
```
This command already exists in the `Setting Shortcuts` section. Please, delete your newly added `Win + I` command.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify your codes.

**Comment by aniketsinha2002** [comment]
@crescentpartha  Sir done! Please review it!!

**Review by crescentpartha — APPROVED** [review]
Approved your changes. Thanks

**Pull request merged**